crates/harness/src/test_runner.rs

```diff
@@ -57,6 +58,11 @@ impl TestRunner {
             .expect("failed to respawn balancer");
     }
 
+    pub async fn is_alive(&mut self) {
+        let ecode = self.child.wait().await.expect("Error: Balancer is not alive");
```

Comment:
`wait()` blocks execution until the process exits. We can't do that because we need to assert that the balancer is alive.

Reply:
@dyc3 I remember you mentioned there should be some function that returns some kind of result. The only other functions I saw that were similar are try_into() and wait_with_output(). I also saw self.child.stderr.is_none(), not sure if relevant.

Reply:
https://docs.rs/tokio/latest/tokio/process/struct.Child.html#method.try_wait